fastlane/lib/fastlane/actions/download_dsyms.rb

```diff
@@ -53,7 +53,7 @@ def self.run(params)
 
         # Write a nice message
         message = []
-        message << "Looking for dSYM files for #{params[:app_identifier]}"
+        message << "Looking for dSYM files for #{platform} #{params[:app_identifier]}"
```

Comment:
To make clearer that the platform name is not part of the app identifier:
```suggestion
        message << "Looking for dSYM files for `#{params[:app_identifier]}` (platform: #{platform})"
```
Is `platform` always set or could it be empty?

Reply:
Platform is required afaik